<launch>
  <rosparam file="$(find ft_calibration)/config/ur/calibration.yaml" command="load"/>
  <rosparam file="$(find ft_calibration)/config/ur/hardware.yaml" command="load"/>
  <node pkg="ft_calibration" name="ft_calibration_node" type="ft_calibration_node"  output="screen" />
</launch>
